core/new-gui/src/app/home/component/login/google-login/google-login.component.ts

```diff
@@ -20,10 +22,11 @@ export class GoogleLoginComponent {
     this.userService
       .googleLogin()
       .pipe(untilDestroyed(this))
-      .subscribe(() => {
-        if (this.userService.getCurrentUser()) {
-          location.href = "dashboard/workflow";
-        }
-      });
+      .subscribe(
+        Zone.current.wrap(() => {
+          // TODO temporary solution, need improvement
```

Comment:
Please add more details about why this is a temporary solution, and what is this solution for. Again, we want the reader of this code understand why we need to add Zone wrapper.